Diagnostics/HealthChecker/Analyzer/Invoke-AnalyzerEngine.ps1

```diff
@@ -1257,7 +1257,7 @@
         }
 
         if ($certificate.PublicKeySize -lt 2048) {
-            $additionalDisplayValue = "It's recommended to use a key size of at least 2048 bit."
+            $additionalDisplayValue = "It's recommended to use a key size of at least 2048 bit"
```

Comment:
Do we need to set it to a variable if it is only used once and doesn't change?

Reply:
Agree. Fixed.